RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml.cs

```diff
@@ -20,23 +20,23 @@ public IndexModel(ILogger<IndexModel> logger)
             _logger = logger;
         }
 
-        public IActionResult OnGet(string search)
+        public JsonResult OnGet(string search)

```

Comment:
A couple of cautions here:
1) This is changing the functionality of the app, from returning an HTML page, to returning a JSON stream.  Perhaps the group wants to have an HTML page.
2) If you're not using the HTML page, then several of the changes to the HTML page above will be inconsequential, as that HTML page will never be viewed.